### DISCUSSION TITLE
Using the new MeterGroup component - where is the MeterItem interface?

### DISCUSSION THREAD

**Discussion by morsagmon** [primefaces/community#1562] [PrimeNG] [answered]
In the new MeterGroup component of PrimeNG there is a mention of the MeterItem type.
Where can I find the interface definition for this MeterItem type, and better yet with a code example of using it (the documentation only show how to call the MeterGroup from the template, not how the class is written)?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for reporting the issue, we'll address it with 17.14.0 release. https://github.com/primefaces/primeng/issues/15284

Until then, here is the meterItem interface

```
interface MeterItem {
    label?: string;
    value?: number;
    color?: string;
    icon?: string;
}

```
